pkg/aws/service.go

```diff
@@ -311,10 +311,10 @@ func Construct(cfg Config) (types.Service, error) {
 	blobRegistryTableCfg := cfg.Config.Copy()
 	blobRegistryTableCfg.Region = cfg.LegacyBlobRegistryTableRegion
 	legacyMigratedShardChecker := NewDynamoMigratedShardChecker(
-		dynamodb.NewFromConfig(storeTableCfg),
-		dynamodb.NewFromConfig(blobRegistryTableCfg),
 		cfg.LegacyStoreTableName,
+		dynamodb.NewFromConfig(storeTableCfg),
 		cfg.LegacyBlobRegistryTableName,
+		dynamodb.NewFromConfig(blobRegistryTableCfg),
```

Comment:
here we were giving `LegacyStoreTableName` in the place of `blobRegistryTableName` and viceversa